.rubocop_todo.yml

```diff
@@ -103,6 +90,7 @@ Style/Documentation:
 # SupportedStyles: always, always_true, never
 Style/FrozenStringLiteralComment:
   Exclude:
+    - '**/*.arb'
```

Comment:
What are `arb` files?

Reply:
(the todo is file completely autogenerated by rubocop. it excludes arb even if we don't have them in the repo).